releasenotes/notes/python-311-support-7f5e6e135fd1e791.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Adds Python 3.11 support, and removes Python 3.7 support. Starting with this release, Qiskit
```

Comment:
I would split out the 3.7 removal into a separate `upgrade` note. Since this is a removal any existing 3.7 users will need to know about.

Reply:
ah right - thanks, good catch.